sysdig/internal/client/v2/allowed_ip_ranges.go

```diff
@@ -0,0 +1,114 @@
+package v2
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+var AllowedIpRangeNotFound = errors.New("IP range not found")
+
+const (
+	AllowedIPRangesPath = "%s/platform/v1/allowed-ip-ranges"
+	AllowedIPRangePath  = "%s/platform/v1/allowed-ip-ranges/%d"
+)
+
+type AllowedIPRangesInterface interface {
+	Base
+	GetAllowedIpRangeById(ctx context.Context, id int) (*AllowedIpRange, error)
```

Comment:
In go we should name this `GetAllowedIPRangeByID` this comment applies for all other methods

Reply:
updated